app/assets/stylesheets/v3/layout_components/sections.scss

```diff
@@ -291,7 +291,7 @@
 
 .main-container {
   // using grid-column in place of large-10 so that we can set the gutters to 0
-  @include grid-column(10,0);
+  @include grid-column(12,0);
```

Comment:
should we try to imitate the width of the marketing site more?

Reply:
I think we had a quick call about this last week and decided to approach this once v2 comes around. The biggest challenge was that we need a full-width page header, but the rest of the page content should be inset with their custom page widths, and that would require custom code in each application.

Reply:
ah okay, sorry I either forgot or wasn't there for that. Sounds good.